kroxylicious-operator/src/main/resources/META-INF/fabric8/virtualkafkaclusters.kroxylicious.io-v1.yml

```diff
@@ -67,6 +67,17 @@ spec:
                         bootstrapAddress:
                           type: string
                           minLength: 1
+                ingresses:
+                  type: array
+                  minItems: 1
```

Comment:
should we allow empty ingresses?

Reply:
Define the use case and the operator behaviour

Reply:
a virtual cluster with no ingresses would have to be omitted from the config, so I think requiring >=1 ingress is reasonable.